server/events/markdown_renderer_test.go

```diff
@@ -60,7 +60,7 @@ func TestRenderErr(t *testing.T) {
 		},
 	}
 
-	r := events.NewMarkdownRenderer(false, false, false, false, false, false, "", "atlantis", false)
+	r := events.NewMarkdownRenderer(false, false, false, false, false, false, "", "atlantis", false, false)
```

Comment:
```suggestion
	r := events.NewMarkdownRenderer(
		false,      // gitlabSupportsCommonMark
		false,       // disableApplyAll
		false,      // disableApply
		false,      // disableMarkdownFolding
		false,      // disableRepoLocking
		false,      // enableDiffMarkdownFormat
		"",         // MarkdownTemplateOverridesDir
		"atlantis", // executableName
		false,      // hideUnchangedPlanComments
		false,      // quietPolicyChecks
	)
```

Can I get you to update these function calls to comment on each flag what it is, here and below, to make the tests more consistent and readable?